CHANGELOG.md

```diff
@@ -20,11 +20,15 @@ general bugfixes
 
 
 ## v2.6.0
+
+#### IMPORTANT
+- Added variable to settings.py to overwrite allowed children of CMSAllinkSEOAccordionPlugin [#76](https://github.com/allink/allink-core/pull/76)
+    - add ```ALLINK_SEOACCORDION_PLUGIN_CHILD_CLASSES = []``` to settings.py
```

Comment:
please add ALLINK_SEOACCORDION_PLUGIN_CHILD_CLASSES to allink_settiings.py with the default value. So we won't have to change anything in settings.py when updating allink-core